assets/js/editor.js

```diff
@@ -502,11 +818,25 @@ class TabBar {
 }
 
 
+/**
+ * Represents the code editor.
+ *
+ * Serves as a convenience wrapper for the base editor.
```

Comment:
* What does "base editor" refer to? The Ace.js editor? If so, say it directly.
* Add some high-level info about how the editor works and how it reuses Ace's features. For example: one Ace `Editor` object, a `Session`s for each tab (is that the way it works? I don't know yet).
* Add some high-level info about how this `Editor` class is supposed to be used.

Reply:
I added an example (in a future commit) to explain how to use the editor and more high level information on how the editor wraps `ace.js`.